core/storage/topic/gae_models.py

```diff
@@ -51,6 +51,10 @@ class TopicModel(base_models.VersionedModel):
     name = ndb.StringProperty(required=True, indexed=True)
     # The canonical name of the topic, created by making `name` lowercase.
     canonical_name = ndb.StringProperty(required=True, indexed=True)
+    # The abbreviated name of the topic.
+    abbreviated_name = ndb.StringProperty(required=True, indexed=True)
+    # The thumbnail of the topic.
+    thumbnail = ndb.StringProperty(indexed=True)
```

Comment:
thumbnail_filename (please fix the comment too)

Reply:
Done.